appdatabase/migrations/sql/1708416025_make_sepolia_default.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE settings RENAME COLUMN is_sepolia_enabled TO is_goerli_enabled;
```

Comment:
@alaibe does this work on iOS? I remember in the past we couldn't do it that way.

Reply:
Yes, iOS gets the following error (on login) `failed to apply status-go/protocol migrations: failed to migrate: file does not exist` @alaibe

Reply:
Alright, i didn't know iOS was special, fixing that